src/components/Accordion/index.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  Accordion as ChakraAccordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  Box,
+} from "@chakra-ui/react"
+
+type AccordionProps = {
+  label: string
+  children: React.ReactNode
+}
+
+const Accordion = ({ label, children }: AccordionProps) => (
+  <ChakraAccordion allowToggle>
+    <AccordionItem border="none">
+      {({ isExpanded }) => (
+        <>
+          <AccordionButton
+            outline="none"
+            border="4px"
+            borderColor="transparent"
+            _hover={{ bg: "primary.lowContrast" }}
+            _expanded={{ bg: "primary.lowContrast" }}
+            _focus={{
+              border: "4px",
+              borderColor: "primary.lowContrast",
+              borderRadius: "md",
+            }}
+            py="2"
+            px={{ base: 2, md: 4 }}
+          >
```

Comment:
All of this styling should still move to a component theme file, as there is an [imported default theme]() that could cause specificity issues.

Also, with the focus styling, it should be applied to the `outline` and not `border`, else you will get a layering when tabbing to the button, which will not comply with the DS.